Smash_Combos.Core/Cqrs/Users/GetUser/GetUserRequestHandler.cs

```diff
@@ -30,7 +30,7 @@ public async Task<GetUserResponse> Handle(GetUserRequest request, CancellationTo
                         .Include(user => user.Comments)
                             .ThenInclude(comment => comment.Reports)
                         .Include(user => user.Infractions)
-                        .Where(user => user.Id == request.UserId)
+                        .Where(user => user.DisplayName == request.DisplayName)
                         .FirstOrDefaultAsync();
```

Comment:
Probably best to use single or default to get sensitive data like a user, we don't want to accidentally return the first if someone does something cheeky. Which is definately possible when using display name as the identifier.